src/pyuvdata/uvdata/uvfits.py

```diff
@@ -408,7 +414,8 @@ def read_uvfits(
             # check if we have an spw dimension
             if vis_hdr["NAXIS"] == 7:
                 self.Nspws = vis_hdr.pop("NAXIS5")
-                self.spw_array = fits_utils._gethduaxis(vis_hdu, 5).astype(np.int64) - 1
+                # By UVFITS definition, this _has_ to be the way this axis is organized
+                self.spw_array = np.arange(self.Nspws)
```

Comment:
I think this is a violation of the FITS standard, but if AIPS does this we shouldn't break on it.